pkg/storaged/mdraid-details.jsx

```diff
@@ -208,6 +191,23 @@ export class MDRaidDetails extends React.Component {
             );
         }
 
+        function fix_bitmap() {
+            return mdraid.SetBitmapLocation(utils.encode_filename("internal"), { });
+        }
+
+        let bitmap_message = null;
+        if (mdraid.Size > 100 * 1024 * 1024 * 1024 &&
+            mdraid.BitmapLocation && utils.decode_filename(mdraid.BitmapLocation) == "none") {
+            bitmap_message = (
+                <Alert isInline variant="warning"
+                       title={_("This RAID array has no bitmap.  A bitmap can reduce sychronization times significantly.")}>
+                    <div className="storage_alert_action_buttons">
+                        <StorageButton onClick={fix_bitmap}>{_("Add a bitmap")}</StorageButton>
+                    </div>
+                </Alert>
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18979-20230630-121722-3b196e14-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)